src/pages/AdminDashboard/roomsActions.ts

```diff
@@ -68,6 +69,9 @@ export const createRoom = (t, room: Room, remindAt: number) => async (
         const { destinations } = room
 
         dispatch(setRoomCreated(roomId, countDestinations(destinations)))
+        if (destinations.length) {
+            dispatch(getUserDetails(t))
```

Comment:
shouldn't we always get the rooms list, but only display the meeting still to happen ?

Reply:
this is only used to refresh quota/usage block (on the right in admin dashboard)